strawberry/enum.py

```diff
@@ -11,45 +12,69 @@ class EnumValue:
     value: Any
 
 
-@dataclasses.dataclass
-class EnumDefinition:
-    name: str
-    values: List[EnumValue]
-    description: Optional[str]
+class StrawberryEnum:
+    def __init__(
+        self,
+        enum: EnumMeta,
+        python_name: str,
+        graphql_name: Optional[str],
+        description: Optional[str],
+    ) -> None:
+        self.enum = enum
+        self.python_name = python_name
+        self._graphql_name = graphql_name
+        self.description = description
+
+    def __call__(self, *args: Any, **kwds: Any) -> Any:
+        return self.enum(*args, **kwds)
+
+    def __getattr__(self, attr):
+        if hasattr(self.enum, attr):
+            return self.enum[attr]
+
+        return super().__getattribute__(attr)
+
+    @property
+    def values(self) -> List[EnumValue]:
+        return [
+            EnumValue(item.name, item.value) for item in cast(Iterable[Enum], self.enum)
+        ]
```

Comment:
Do we need `EnumValue` anymore? Or can the schema converter directly grab the values?